ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-2024a.yaml

```diff
@@ -0,0 +1,471 @@
+- context_dir: runtimes/minimal/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: base-ubi8-python-3.8
+  to: runtime-minimal-ubi8-python-3.8
+- context_dir: runtimes/datascience/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: base-ubi8-python-3.8
+  to: runtime-datascience-ubi8-python-3.8
+- context_dir: runtimes/pytorch/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: base-ubi8-python-3.8
+  to: runtime-pytorch-ubi8-python-3.8
+- context_dir: runtimes/tensorflow/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: cuda-ubi8-python-3.8
+  to: runtime-cuda-tensorflow-ubi8-python-3.8
+- context_dir: base/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  to: base-ubi9-python-3.9
+- context_dir: jupyter/minimal/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.9
+  to: jupyter-minimal-ubi9-python-3.9
+- context_dir: jupyter/datascience/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: jupyter-minimal-ubi9-python-3.9
+  to: jupyter-datascience-ubi9-python-3.9
+- context_dir: cuda/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.9
+  to: cuda-ubi9-python-3.9
+- context_dir: jupyter/minimal/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: cuda-ubi9-python-3.9
+  to: cuda-jupyter-minimal-ubi9-python-3.9
+- context_dir: jupyter/datascience/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: cuda-jupyter-minimal-ubi9-python-3.9
+  to: cuda-jupyter-datascience-ubi9-python-3.9
+- context_dir: jupyter/tensorflow/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: cuda-jupyter-datascience-ubi9-python-3.9
+  to: cuda-jupyter-tensorflow-ubi9-python-3.9
+- context_dir: jupyter/pytorch/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: cuda-jupyter-datascience-ubi9-python-3.9
+  to: jupyter-pytorch-ubi9-python-3.9
+- context_dir: jupyter/trustyai/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: jupyter-datascience-ubi9-python-3.9
+  to: jupyter-trustyai-ubi9-python-3.9
+- context_dir: runtimes/minimal/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.9
+  to: runtime-minimal-ubi9-python-3.9
+- context_dir: runtimes/datascience/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.9
+  to: runtime-datascience-ubi9-python-3.9
+- context_dir: runtimes/pytorch/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.9
+  to: runtime-pytorch-ubi9-python-3.9
+- context_dir: runtimes/tensorflow/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: cuda-ubi9-python-3.9
+  to: runtime-cuda-tensorflow-ubi9-python-3.9
+- context_dir: codeserver/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.9
+  to: codeserver-ubi9-python-3.9
+- context_dir: base/c9s-python-3.9
+  dockerfile_path: Dockerfile
+  to: base-c9s-python-3.9
+- context_dir: codeserver/c9s-python-3.9
+  dockerfile_path: Dockerfile
+  from: base-c9s-python-3.9
+  to: codeserver-c9s-python-3.9
```

Comment:
lets remove centos based code-server build , as we build ubi based images for code-server.

Reply:
Good catch , removed.